Author: José Manuel Santamaría Lema <panfaust@gmail.com>
Description: This patch disables Get Hot New Stuff support because it would
 work only with GHNS 3 which is not used by libkdegames 4.4.x.
 This pacth should be removed after uploading kdegames 4.5.x or later to
 unstable.
Forwarded: no
--- a/src/knights.cpp
+++ b/src/knights.cpp
@@ -255,7 +255,7 @@
 #endif
         dialog->addPage ( generalSettingsDlg, i18n ( "General" ), QLatin1String ( "games-config-options" ) );
         connect ( dialog, SIGNAL ( settingsChanged ( QString ) ), m_view, SLOT ( settingsChanged() ) );
-        QWidget* themeDlg = new KGameThemeSelector ( dialog, Settings::self(), KGameThemeSelector::NewStuffEnableDownload );
+        QWidget* themeDlg = new KGameThemeSelector ( dialog, Settings::self(), KGameThemeSelector::NewStuffDisableDownload );
         dialog->addPage ( themeDlg, i18n ( "Theme" ), QLatin1String ( "games-config-theme" ) );
         dialog->setAttribute ( Qt::WA_DeleteOnClose );
         dialog->show();
